lib/android_build/app/build.gradle

```diff
@@ -30,14 +30,19 @@ android {
             version "3.10.2"
         }
     }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }

```

Comment:
Is this dependency coming for okhttp ? As I understand there are small number of android devices on java 6/7, the build will fail for those?

Reply:
latest android stuido uses java 11 by default, so it failed to compile, this tells it use 1.8 like in maesdk library